app/resources/en/translation/coapplicant/allagreedpage.json

```diff
@@ -5,8 +5,6 @@
   "subHeader": "What happens next",
   "paragraph2": "The person responsible for making this application will receive the grant of probate within 20 working days of the probate registry receiving all the necessary paperwork.",
   "paragraph3": "Contact the executor who&rsquo;s responsible for this application if you need more information.",
-  "paragraph4": "You&rsquo;ll need to sign a photocopy of the will if you haven&rsquo;t already.",
-  "paragraph4-codicils": "You&rsquo;ll need to sign a photocopy of the will and any codicils if you haven&rsquo;t already.",
```

Comment:
Please update the codicils/no codicils component tests for this page.

`test/component/coapplicant/testCoApplicantAllAgreePage.js`